homeassistant/components/uptime_kuma/config_flow.py

```diff
@@ -0,0 +1,77 @@
+"""Config flow to configure the Uptime Kuma integration."""
+from uptime_kuma_monitor import UptimeKumaError, UptimeKumaMonitor
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_PASSWORD,
+    CONF_PORT,
+    CONF_USERNAME,
+    CONF_VERIFY_SSL,
+)
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import DOMAIN
+
+
+class UptimeKumaFlowHandler(ConfigFlow, domain=DOMAIN):
+    """Handle an Uptime Kuma config flow."""
+
+    VERSION = 1
+
+    _hassio_discovery = None
+
+    async def _show_setup_form(self, errors=None) -> FlowResult:
+        """Show the setup form to the user."""
+        return self.async_show_form(
+            step_id="user",
+            data_schema=vol.Schema(
+                {
+                    vol.Required(CONF_HOST): str,
+                    vol.Optional(CONF_PORT, default=443): vol.Coerce(int),
+                    vol.Required(CONF_USERNAME): str,
+                    vol.Required(CONF_PASSWORD): str,
+                    vol.Optional(CONF_VERIFY_SSL, default=True): bool,
+                }
+            ),
```

Comment:
Consider moving the schema definition to a constant at the top, outside of the UptimeKumaFlowHandler class.
`STEP_USER_DATA_SCHEMA = vol.Schema(...`

```suggestion
            data_schema=STEP_USER_DATA_SCHEMA,
```